### DISCUSSION TITLE
course/starbook/word-delete

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#117] [General]
# course/starbook/word-delete

通过API实现单词删除功能，使用DELETE方法请求删除指定ID的单词。Logic层根据用户ID和单词ID进行数据库删除操作，控制器层负责接收请求并调用逻辑删除单词。提供了详细的接口测试步骤用于验证功能实现。

https://goframe.org/course/starbook/word-delete

**Comment by notmaster-C** [comment]
建议加入软删除

**Comment by chaogenb110** [comment]
c.users.GetUid(ctx) 这个GetUid 在文档里没有体现出来吧？我翻了好多次都没有看到，我每次都是在logic文件里面去解析一次token，然后用token去获取用户id，但是感觉很麻烦，这个getUid是否就是封装成一个公共的func，来进行调用？
